main/src/ca/uwaterloo/flix/language/phase/unification/BoolUnification.scala

```diff
@@ -161,23 +161,8 @@ object BoolUnification {
     } else if (alg.isFalse(f)) {
```

Comment:
This method can now be removed, right? We can just call `alg.satisfiable` directly?